src/custom/utils/appzi.ts

```diff
@@ -67,24 +67,31 @@ export function updateAppziSettings({ data = {}, userId = '' }: AppziSettings) {
 }
 
 export function openFeedbackAppzi() {
-  console.debug(`Showing appzi feedback. Always!`)
   window.appzi?.openWidget(FEEDBACK_KEY)
 }
 
-function restyleAppziNps() {
-  // Add a unique class based on NPS_KEY
-  const appziRoot = document.querySelector("div[id^='appzi-wfo-']")
-  if (appziRoot) {
-    appziRoot.classList.add(`appzi-nps-${NPS_KEY}`)
+let appziRestyleApplied = false
+function applyOnceRestyleAppziNps() {
+  if (!appziRestyleApplied) {
+    appziRestyleApplied = true
+    // Make sure we apply the re-style once the appzi NPS is loaded
+    onceAppziEvent(restyleAppziNps)
   }
 }
 
-export function openNpsAppzi() {
-  console.debug(`Showing appzi NPS. Always!`)
-
-  // Make sure we apply the re-style once the appzi NPS is loaded
-  onceAppziEvent(restyleAppziNps)
+function restyleAppziNps(event: any) {
+  if (event && event.type && event.type === 'open-survey') {
```

Comment:
NItpick: Second condition not needed because `undefined !== 'open-survey'`, in case `event.type` isn't set

Reply:
https://github.com/cowprotocol/cowswap/pull/883